### ISSUE TITLE
Same message for different alarms

### ISSUE THREAD

**Issue by ach80** [OpenSeizureDetector/Android_Pebble_SD#20]
Currently message for every kind of alarm is the same. It seems to be a much expected feature to have different messages and SMS alerts for a seizure, heart rate, and fall alarm.

**Comment by jones139** [comment]
Yes, can definitely be done - will mean having three different sets of translations though :).

**Comment by ach80** [comment]
No prob here :)
